Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -417,9 +417,9 @@
 				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = "iOS Example/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				OTHER_SWIFT_FLAGS = "$(inherited) -D DEBUG";
```

Comment:
Yes, I expanded the example app to have different logging levels for DEBUG vs non DEBUG builds to better reflect real-world usage.